CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# dbt_tiktok_ads v0.7.0
+[PR #22](https://github.com/fivetran/dbt_tiktok_ads/pull/22) includes the following updates:
+
+## Breaking Changes
+- **Upstream Breaking Change:** In the [July 2023 TikTok Ads connector update](https://fivetran.com/docs/connectors/applications/tiktok-ads/changelog#july2023) for the `ADGROUP_HISTORY` table, the `age` column was renamed to `age_groups`.
+  - This change primarily affects the upstream `stg_tiktok_ads__ad_group_history` model in `dbt_tiktok_source`. The `age_groups` column was not previously used in any downstream models within `dbt_tiktok_ads`, so this change impacts only the upstream staging model.
+  - Previously, we coalesced the `age` and `age_groups` columns in the `stg_tiktok_ads__ad_group_history` model to accommodate connectors using the old naming convention. However, due to inconsistent data types, this approach is no longer viable.
+  - As a result, the coalesced field has been removed in favor of the `age_groups` column.
+  - If necessary, you can populate historical data in the `age_groups` column by performing a resync of the `ADGROUP_HISTORY` table, as TikTok provides all data regardless of the previous sync state.
+  - For more details, see the [DECISIONLOG entry](https://github.com/fivetran/dbt_tiktok_ads_source/blob/main/DECISIONLOG.md).
+
+
```

Comment:
Should we note the creation of the DECISIONLOG in a Documentation section here?

Reply:
Along with the stuff about metrics across different grains

Reply:
Added.